app/controllers/application_controller.rb

```diff
@@ -2,6 +2,7 @@ class ApplicationController < ActionController::Base
   include Pundit
 
   before_action :set_paper_trail_whodunnit
+  before_action :set_requsted_url
```

Comment:
typo: can we make this `set_requested_url` or maybe make it `return_to_url`?

the typo was copied very consistently